validator/project_validator.go

```diff
@@ -1403,30 +1404,44 @@ func checkTaskRuns(project *model.Project) ValidationErrors {
 						bvtu.Name, bvtu.Variant),
 				})
 			}
+			for _, requester := range bvtu.AllowedRequesters {
+				if requester.Validate() != nil {
+					errs = append(errs, ValidationError{
+						Level: Warning,
+						Message: fmt.Sprintf("task '%s' in build variant '%s' specifies invalid allowed_requester '%s'",
+							bvtu.Name, bvtu.Variant, requester),
+					})
+				} else {
+					hasValidAllowedRequester = true
+				}
+			}
+		} else {
```

Comment:
Can remove this else block and initialize the variable as `hasValidAllowedRequester := len(bvtu.AllowedRequesters) == 0`

Reply:
Sure, done.